### ISSUE TITLE
[Feedback] Regarding the AI and Secrets

### ISSUE THREAD

**Issue by SirFatCat** [demilich1/metastone#42]
Something that I've been noticing as I've been playing during Play Mode is that whenever I play a secret, the AI just kind of locks up in regards to attacking minions even when they have answers to the secret. They just kind of stop playing and attack face until the secret is gone, and it makes the AI feel much less like you're playing against someone who plays around various situations in which a secret could be played and much more like against a bot who knows what you've played and just doesn't want to play anything to counteract it. 

A normal player would attack a 1-1 when they have four 1-1 minions against a Mage, but the AI doesn't do anything because of the possibility of Duplicate/Effigy. It just hits the face. If you have a taunt, they don't do anything at all unless they have the resources to get rid of the taunt and more. Usually this means you win because you get like 5 turns of nothing on the end of the other player. 

I don't expect the AI to be top notch or anything of the sort, I just wanted to post some feedback on the AI that I feel really sticks out. If the AI was more fleshed out in regards to secrets, I would consider it to be pretty good for the simulator, even though it still has small issues here and there, of course.

**Comment by demilich1** [comment]
True, in some situations the AI behaves somewhat awkward and in other situations it acts just plain stupid. There may be some quick fixes for the secrets, however I think to really improve the AI it has to be re-written from scratch. Once we got the card mechanics sorted, I think someone will take care of that. I already received emails from three different guys who want to use MetaStone for their thesis at university.

So basically I agree with your feedback, but I think it will still take some time till we can realize a better AI.
